File: ssaExample/types_Info.java
import java.util.Map;

public class types_Info {

	public Map<Object, types_TypeAndValue> Types;
	public Map<ast_Ident, types_Instance> Instances;
	public Map<ast_Ident, Object> Defs;
	public Map<ast_Ident, Object> Uses;
	public Map<Object, Object> Implicits;
	public Map<ast_SelectorExpr, types_Selection> Selections;
	public Map<Object, types_Scope> Scopes;
	public types_Initializer[] InitOrder;
}

File: ssaExample/types_Info.kt
class types_Info {

	var Types: Map<Any, types_TypeAndValue>? = null
	var Instances: Map<ast_Ident, types_Instance>? = null
	var Defs: Map<ast_Ident, Any>? = null
	var Uses: Map<ast_Ident, Any>? = null
	var Implicits: Map<Any, Any>? = null
	var Selections: Map<ast_SelectorExpr, types_Selection>? = null
	var Scopes: Map<Any, types_Scope>? = null
	var InitOrder: Array<types_Initializer>? = null
}
